docs/testing/developer-smoke-test-cases.md

```diff
@@ -53,6 +54,38 @@ We currently do **not** provide support for or test against the following enviro
 + Clusters with 1 or more Windows workers interacting with **multiple** domains at once, either joined or un-joined
 + Clusters interacting with **multiple** Active Directory instances at once
 
+## Configure RBAC for your service account
+
+In order to test the Account Provider you must first install the GMSA Web-hook, which will provide the `GMSACredentialSpec` CRD  expand the contents of the resource onto pods. The web-hook chart will deploy a purpose built `ClusterRole` which permits access to all `GMSACredentialSpec` resources. However, the default `ClusterRoleBinding` deployed by the GMSA Web-hook chart only grants access to these resources to the service account used by the GMSA web-hook. In order to create workloads which utilize a `GMSACredentialSpec`, but run under a different service account, an additional `Role` and `RoleBinding` needs to be created for the service account you intend to use for testing.
```

Comment:
For now, this is fine, but I think we should just refactor this developer-smoke-test-cases document to just reference https://github.com/rancher/windows/blob/main/docs/terraform/gmsa/environment_setup.md.

We don't want to have two different documented developer workflows for provisioning gMSA setups.